packages/plugins/i18n/server/src/controllers/content-types.ts

```diff
@@ -16,7 +16,7 @@ const getFirstLevelPath = map((path: string) => path.split('.')[0]);
 const controller = {
   async getNonLocalizedAttributes(ctx) {
     const { user } = ctx.state;
-    const { model, id, locale } = ctx.request.body;
+    const { model, id, locale } = ctx.request.body as any; // TODO: add correct typings
```

Comment:
I guess this shouldn't really be part (as any other change in i18n types) of the PR and will be fixed directly in main. Do we need to keep those changes anyway?

Reply:
I'll revert them and update from v5/main now that it's apparently fixed there

Reply:
Note on this: adding @types/koa-bodyparser to graphql caused it to start working throughout the monorepo, switching the type of body from 'any' to 'unknown'. So I had to "fix" CTB and i18n so that ts:test:back will pass. Functionally there's no difference, just allowing body to be treated as `any` as it was before this PR (and in the future, new code should treat it as unknown because it is)